.github/CONTRIBUTING.rst

```diff
@@ -195,33 +195,23 @@ account. To set this up please go to this
-| ``mock_online``   | It runs the online tests using mocked information. Does not require user credentials.                              | ``not uc_available``  | ``skip_online = False``                          |
-+-------------------+--------------------------------------------------------------------------------------------------------------------+-----------------------+--------------------------------------------------+
-| ``run_slow``      | It runs tests tagged as *slow*.                                                                                    | ``False``             |                                                  |
-+-------------------+--------------------------------------------------------------------------------------------------------------------+-----------------------+--------------------------------------------------+
-| ``rec``           | It records the remote requests. It requires user credentials.                                                      | ``False``             | ``skip_online = False``                          |
-|                   |                                                                                                                    |                       | ``run_slow = False``                             |
-+-------------------+--------------------------------------------------------------------------------------------------------------------+-----------------------+--------------------------------------------------+
++-----------------+--------------------------------------------------------------------------------------------------------------------+-----------+
+|  Option         | Description                                                                                                        | Default   |
++=================+====================================================================================================================+===========+
+| ``skip_online`` | Skips tests that require remote requests (also, no mocked information is used). Does not require user credentials. | ``False`` |
++-----------------+--------------------------------------------------------------------------------------------------------------------+-----------+
+| ``run_slow``    | It runs tests tagged as *slow*.                                                                                    | ``False`` |
++-----------------+--------------------------------------------------------------------------------------------------------------------+-----------+
 
 It is possible to provide more than one option separated with commas.
-The order of precedence in the options is right to left. For example,
-``QISKIT_TESTS=skip_online,rec`` will set the options as
-``skip_online == False`` and ``rec == True``.
+The order of precedence in the options is right to left.
```

Comment:
Is this still necessary now that we only have `skip_online` and `run_slow`?

Reply:
Not really, I'll just remove it.